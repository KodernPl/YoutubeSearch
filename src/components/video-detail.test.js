import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import VideoDetail from './video-detail';

const renderComponent = () => {
  return shallow((
    <VideoDetail
      video={{
        id: { videoId: 'ID' },
        snippet: {
          title: 'TITLE',
          description: 'DESC',
          publishedAt: 'PUB',
        },
      }}
    />
  ));
};

test('renders correctly', () => {
  const component = renderComponent();

  expect(toJson(component)).toMatchSnapshot();
});
