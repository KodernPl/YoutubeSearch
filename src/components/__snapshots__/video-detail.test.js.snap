// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <video-detail__Video>
    <iframe
      src="https://www.youtube.com/embed/ID"
      title="video"
    />
  </video-detail__Video>
  <video-detail__Details>
    <video-detail__Title>
      TITLE
    </video-detail__Title>
    <video-detail__Paragraph>
      DESC
    </video-detail__Paragraph>
    <video-detail__Paragraph>
      Published at: 
      PUB
    </video-detail__Paragraph>
  </video-detail__Details>
  <video-detail__Line />
</Fragment>
`;
